.github/workflows/tee_check.yml

```diff
@@ -0,0 +1,97 @@
+name: Runtime and cross_tee testing
+# push, pull_request,
+on:  [push, pull_request]
+
+jobs:
+    null_instance_check:
+      # differetn null instance combination  
+      strategy:
+        matrix:
+          tag: [anolis8.6, ubuntu20.04]
+
+      # if only specify a container image, can omit the `image` keyword 
+      # get the image from the docker hub 
+      container: runetest/compilation-testing:${{ matrix.tag }}
+  
+      # Use GitHub-hosted runner Ubuntu 20.04
+      runs-on: ubuntu-20.04
+      
+      defaults: 
+        run:
+          shell: bash
+          working-directory: ${{ github.workspace }}
+      env: 
+        HOME: /root
+      #todo:  use cache to boost the workflow
+      steps:
+        - name: Checkout Code
+          uses: actions/checkout@v2
+
+
+        - name: Build and install "host" mode
+          run: |
+            source /root/.bashrc
+            cmake -DBUILD_SAMPLES=on -H. -Bbuild
+            make -C build install
+
+        # install dependencies
+        - if: ${{matrix.tag == 'ubuntu20.04'}}
+          run: apt-get install lsof
+        - if: ${{matrix.tag == 'anolis8.6'}}
+          run: yum install lsof -y
+
+        - id: rand_port_1
+          uses: ./.github/actions/generate-random-port
+
+        #         TLS Wrapper | Attester | Verifier | Crypto Wrapper
+        # server: null        | null     | null     | null              
+        # client: null        | null     | null     | null                
+        - name: Null Instance test 1
```

Comment:
please remove line 46-48, and change line 49 to `Name: Host mode with nullattester/nullverifier/nulltls/nullcrypto` so that we can know the test case combination in github action page.